.eslintrc.json

```diff
@@ -1,13 +1,13 @@
 {
   "extends": [
-    "plugin:mattermost/react",
     "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended"
+    "plugin:@typescript-eslint/recommended",
+    "plugin:mattermost/react"
   ],
   "parser": "@typescript-eslint/parser",
   "plugins": [
-    "mattermost",
-    "@typescript-eslint"
+    "@typescript-eslint",
+    "mattermost"
```

Comment:
reason for the change?

Reply:
The rules from the 'mattermost' eslint plugins were getting overridden by the '@typescript-eslint' plugin.  To remediate to it, I have moved the 'mattermost' plugin down the array.